### DISCUSSION TITLE
Is it possible to login to Youtube with my account?

### DISCUSSION THREAD

**Discussion by Drallas** [FreeTubeApp/FreeTube#4076] [Q&A] [answered]
I like the idea of FreeTube, but I still like to log on to YT and access my watch lists and other playlists. For me this missing keeps me in the browser with an addblocker.

**Comment by absidue** [comment, marked as answer]
No, using YouTube with a Google account makes you infinitely more trackable for YouTube, which defeats the entire purpose of FreeTube, which is to store your data locally, respect your privacy and let you use YouTube as anonymously as we can achieve.

**Comment by Drallas** [comment]
> No, using YouTube with a Google account makes you infinitely more trackable for YouTube, which defeats the entire purpose of FreeTube, which is to store your data locally, respect your privacy and let you use YouTube as anonymously as we can achieve.

I understand, but some people care less about that and just want to block the adds, and still use the rest of their Youtube Account. 

Would be nice as a user to have that choice, and use it or not!

**Reply by absidue** [reply]
Again, that goes against the primary goal of FreeTube, violating that would undermine user trust. If the only thing you want from FreeTube is the ability to block ads, I recommend sticking to the official clients with an ad-blocker.

**Reply by Drallas** [reply]
No problem. 

Well yes that's how I would like to use Freetube, currently it's more like a ShadowTube. 😶‍🌫️

Not for me then!
